kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopFsDelegationTokenProvider.scala

```diff
@@ -63,19 +63,26 @@ class HadoopFsDelegationTokenProvider extends HadoopDelegationTokenProvider with
 
   override def obtainDelegationTokens(owner: String, creds: Credentials): Unit = {
     doAsProxyUser(owner) {
-      val fileSystems = fsUris.map(FileSystem.get(_, hadoopConf)).toSet
+      val fileSystems = fsUris.map { uri =>
+        uri -> FileSystem.get(uri, hadoopConf)
+      }
```

Comment:
This changes the semantics, omitting "dedup"

Reply:
thanks